addons/website_customer/controllers/main.py

```diff
@@ -69,7 +63,7 @@ def customers(self, country_id=0, page=0, country_name='', tag_id=0, **post):
         })
 
         # search customers to display
-        partner_count = partner_obj.search_count(cr, openerp.SUPERUSER_ID, domain, context=request.context)
+        partner_count = PartnerSudo.search_count(domain)
```

Comment:
We can put search here, as we are already using search at line 77 and then len(partners)